plex.json

```
@@ -1,59 +1,66 @@
-        "website": "https://hub.docker.com/r/linuxserver/plex/",
-        "version": "1.0"
-    }
-}
+{
+  "Plex-QuickSync": {
+    "containers": {
+      "plex-linuxserver.io-qs": {
+        "image": "linuxserver/plex",
+        "opts": [
+          [
+            "--net=host",
+            ""
+          ]
+        ],
+        "launch_order": 1,
+        "devices": {
+          "dev/dri": {
+            "description": "optional: Device Path for enabling Intel QuickSync (check CPU documentation at at ark.intel.com for availability) (/dev/dri). Leave blank if not needed.",
+            "label": "QuickSync device"
+          }
+        },
+        "ports": {
+          "32400": {
+            "description": "WebUI port. Suggested default: 32400",
+            "host_default": 32400,
+            "label": "WebUI port",
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/config": {
+            "description": "Choose a Share for Plex configuration. E.g.: create a Share called plex-config for this purpose alone. It is important that the owner/group specified later have full access to the share (R/W/E)",
+            "label": "Config Storage"
+          },
+          "/data": {
+            "description": "Choose a Share for Plex content(your media). E.g.: create a Share called plex-data for this purpose alone. You can also assign other media Shares on the system after installation. This share needs to at least have Read/Execute access for the User group specified later",
+            "label": "Data Storage"
+          }
+   
+        },
+        "environment": {
+          "VERSION": {
+            "description": "Choose version of plex: Unless you know which version to try, just type latest",
+            "label": "VERSION",
+            "index": 1
+          },
+          "PUID": {
+            "description": "Enter a valid UID to run Plex with. It must have full permissions to all Shares mapped in the previous step.",
+            "label": "UID",
+            "index": 2
+          },
+          "PGID": {
+            "description": "Enter a valid GID to use along with the above UID. It (or the above UID) must have full permissions to all Shares mapped in the previous step.",
+            "label": "GID",
+            "index": 3
+          }
+        }
+      }
+    },
+    "description": "Plex media server by Linuxserver.io - with added (optional) Intel QuickSync hardware transcoding (check your processor's documentation).",
+    "more_info": "<h4>Adding more media to Plex.</h4><p>You can add more Shares later to Plex from the settings wizard of this Rock-on. If you have maintained a QuickSync Device during the initial setup, you need to add a transcoding share (preferrably on an SSD). Then, from Plex WebUI, you can update the transcode directory setting and for new media shares re-index your library. Visit https://hub.docker.com/r/linuxserver/plex for a description of each option.</p>",

```

Further develop instructions:
```suggestion
    "more_info": "<h4>Adding more media to Plex</h4><p>First, add the share(s) containing your additional media to this rock-on using its settings wizard. Then, from Plex webUI, you can add this/these share(s) and re-index your library. See <a href='http://rockstor.com/docs/docker-based-rock-ons/plex-media-server.html#adding-shares-to-plex' target='_blank'>Rockstor's documentation</a> for detailed instructions.</p> <h4>Hardware transcoding using Intel QuickSync</h4>If you have a compatible processor and filled the <em>QuickSync device</em> field during the rock-on installation, please note that you also need to add a dedicated share (named <em>plex-transcoding</em>, for instance) mapped as <code>/transcode</code> that will be used for transcoding only. Then, from Plex WebUI, you can update the <strong>transcode</strong> directory setting.</p>",
```